### ISSUE TITLE
Wrong query params when parsing query with array params

### ISSUE THREAD

**Issue by MrCheater** [reimagined/resolve#1285]
**Describe the bug**
Wrong query params when parsing query with array params

**?a[]=1&a[]=2**

EXPRESS
```
{
  a: [
    "1",
    "2"
  ]
}
```

AWS
```
event.queryStringParameters: { 'a[]': '2' }
event.multiValueQueryStringParameters: { 'a[]': [ '1', '2' ] }
```

**?a=1&a=2**

EXPRESS
```
{
  a: [
    "1",
    "2"
  ]
}
```

AWS
```
event.queryStringParameters: { a: '2' }
event.multiValueQueryStringParameters: { a: [ '1', '2' ] }
```

**?a=[1,2]**

EXPRESS
```
{
  a: "[1,2]"
}
```

AWS
```
event.queryStringParameters: { a: '[1,2]' }
event.multiValueQueryStringParameters: { a: [ '[1,2]' ] }
```

**?a=1,2**

EXPRESS
```
{
  a: "1,2"
}
```

AWS
```
event.queryStringParameters: { a: '1,2' }
event.multiValueQueryStringParameters: { a: [ '1,2' ] }
```

**?a[]=1,2**

EXPRESS
```
{
  a: [
    "1,2"
  ]
}
```

AWS
```
event.queryStringParameters: { 'a[]': '1,2' }
event.multiValueQueryStringParameters: { 'a[]': [ '1,2' ] }
```

**?a[]=1**

EXPRESS
```
{
  a: [
    "1"
  ]
}
```

AWS
```
event.queryStringParameters: { 'a[]': '1' }
event.multiValueQueryStringParameters: { 'a[]': [ '1' ] }
```

**Comment by alexslavr** [comment]
+1 
Resource unavailable error while loading read-model state with array query params (?a[]=1&a[]=2) in the cloud

**Comment by MrCheater** [comment]
#1348

**Comment by MrCheater** [comment]
Fixed in version 0.23.1
